README.md

```diff
@@ -168,8 +168,8 @@ return [
     ],
     'validators' => [
         'factories' => [
-            Validator\DbRecordExists::class => Validator\Db\RecordExistsFactory::class,
-            Validator\DbNoRecordExists::class => Validator\Db\NoRecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbRecordExists' => Validator\Db\RecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbNoRecordExists' => Validator\Db\NoRecordExistsFactory::class,
```

Comment:
@akrabat, if you check this migration : https://github.com/zfcampus/zf-content-validation/pull/67/files you will notice the introduction of this change, which means the use of: ```::class``` to get the class name, for me this code is not correct because we don't have any class name called (DbNoRecordExists nor DbRecordExists) in this namespace. so for this mapping we need to keep it, juste like before!